File: 11challenge/main.r
equation <- function(a,b,c){
  delta <- (b^2) - (4*a*c)
  equation_string <- paste(a,"x^2 +",b,"x +",c," = 0")
  list(equation = equation_string, delta = delta)
}

eq <- equation(as.numeric(readline("Enter the value of A: ")),
               as.numeric(readline("Enter the value of B: ")),
               as.numeric(readline("Enter the value of C: "))
              )

print(eq)

File: 11challenge/main.py
from math import sqrt

class Equation:
    def __init__(self, a:float, b:float, c:float):
        self.a = a
        self.b = b
        self.c = c
    
    def delta(self):
        return (self.b ** 2) - (4 * self.a * self.c)
    
    def __str__(self):
        return f'Equation is: {self.a}x^2 + {self.b}x + {self.c} = 0, Delta = {self.delta()}'
    

eq = Equation(a=float(input("Enter the value of A: ")),
              b=float(input("Enter the value of B: ")),
              c=float(input("Enter the value of C: "))
             )

print(eq)
